### ISSUE TITLE
Can't see personal wall content that has been bookmarked

### ISSUE THREAD

**Issue by Reuns** [humhub/content-bookmarks#6]
Content in personal wall can be bookmarked, but when you check your bookmark you can't see them.

Humhub 1.8.1
PHP 7.4

**Comment by yurabakhtin** [comment]
@Reuns Could you please tell us step by step what you do and what you expect to see?
Because I tested this so if I save a content as bookmark then I can see it:

<img width="1151" alt="test_bookmark" src="https://user-images.githubusercontent.com/6995524/112796596-df2ef480-9072-11eb-9d85-65c3bb2f7c2b.png">

**Comment by Reuns** [comment]
I made a video (no mic, poor english).
I add someone's content in bookmark.
2 content :
- 1 from a space
- 1 from his personnal wall

I'm verifying that the content are bookmarked (through the "Remove from bookmark" mention)

When I check my saved bookmark, I can see the content from a space, but not the one from the personnal wall.
I expect to see the 2 content


https://user-images.githubusercontent.com/19529531/112818166-861b8c80-9083-11eb-9117-c7af450b5eba.mp4

Does it helps you ?

**Comment by Reuns** [comment]
Oh ... and by the way, as you can see in the clip, I don't have the notification at the bottom of the screen when I add a new content in bookmark

**Comment by yurabakhtin** [comment]
@Reuns Thank you for the video with details. I guess one content is private and you are friend of the user "Sawys". We have implemented a fix for this case in PR https://github.com/humhub/content-bookmarks/pull/8

> I don't have the notification at the bottom of the screen when I add a new content in bookmark

This notification has been merged recently only today here https://github.com/humhub/content-bookmarks/commit/7b9e7b11092c11a53ee9a07fd13560475cc7bad2.
